certified-connectors/ZellisIntelligencePlatform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1063 @@
+        "parameters": [
+          {
+            "x-ms-summary": "Get Worker Posts",
+            "description": "Get Worker Posts from ResourceLink",
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "x-ms-summary": "Get Worker Posts",
+            "description": "Get Worker Posts from ResourceLink",
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ]
+      }
+    },
+    "/notificationhub/v1/subscription": {
+      "x-ms-notification-content": {
+        "description": "WebHook Response",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "specversion": {
+              "type": "string",
+              "description": "specversion"
+            },
+            "id": {
+              "type": "string",
+              "description": "id"
+            },
+            "source": {
+              "type": "string",
+              "description": "source"
+            },
+            "type": {
+              "type": "string",
+              "description": "type"
+            },
+            "time": {
+              "type": "string",
+              "description": "time"
+            },
+            "data": {
+              "type": "object",
+              "properties": {
+                "Action": {
+                  "type": "string",
+                  "description": "Action"
+                },
+                "Entity": {
+                  "type": "string",
+                  "description": "Entity"
+                },
+                "EntityId": {
+                  "type": "string",
+                  "description": "EntityId"
+                },
+                "EntityPath": {
+                  "type": "string",
+                  "description": "EntityPath"
+                },
+                "Payload": {
+                  "type": "object",
+                  "properties": {
+                    "ParameterFive": {
+                      "type": "string",
+                      "description": "ParameterFive"
+                    },
+                    "OperatorSelectionNo": {
+                      "type": "string",
+                      "description": "OperatorSelectionNo"
+                    },
+                    "EventUUID": {
+                      "type": "string",
+                      "description": "EventUUID"
+                    },
+                    "ConnectionType": {
+                      "type": "string",
+                      "description": "ConnectionType"
+                    },
+                    "ParameterOne": {
+                      "type": "string",
+                      "description": "ParameterOne"
+                    },
+                    "ParameterFour": {
+                      "type": "string",
+                      "description": "ParameterFour"
```

Comment:
@basha-k All the description values of properties should be free from grammatical and spell errors. Please  provide valid description values for all the proeprties.

Reply:
These properties are auto generated by the 'paconn download' tool.

Reply:
@basha-k Yes, if you do not have valid descriptions from service, paconn tool would generate as per the property names. 
All the description values of properties should be free from grammatical and spell errors. Please provide valid description values for all the properties.